ext/date/date_core.c

```diff
@@ -7192,13 +7212,15 @@ strftimev(const char *fmt, VALUE self,
 
 /*
  * call-seq:
- *    d.asctime  ->  string
- *    d.ctime    ->  string
+ *   asctime -> string
  *
- * Returns a string in asctime(3) format (but without "\n\0" at the
- * end).  This method is equivalent to strftime('%c').
+ * Equivalent to #strftime with argument <tt>'%a %b %e %T %Y'</tt>
+ * (or its {shorthand form}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html#label-Shorthand+Conversion+Specifiers]
+ * <tt>'%c'</tt>);
  *
- * See also asctime(3) or ctime(3).
+ *   Date.today.asctime # => "Fri Jul 29 00:00:00 2022"
```

Comment:
For all these examples, we should use `Date.new` to make it reproducible for the reader.

Reply:
Done (8 places).